packages/ketcher-react/src/script/ui/views/toolbars/FloatingTools/FloatingTools.container.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  FloatingTools,
+  type FloatingToolsCallProps,
+  type FloatingToolsProps
+} from './FloatingTools'
+
+import { connect } from 'react-redux'
+import { Dispatch } from 'redux'
+import type RotateController from 'src/script/editor/tool/rotate-controller'
+import { onAction } from '../../../state'
+
+const mapStateToProps = (state: any): FloatingToolsProps => {
+  const rotateController: RotateController | undefined =
+    state.editor?.rotateController
```

Comment:
I'm not sure that this is a good practice to use a mutable object (`state.editor`) as a source of reactions.
The `state.editor` is probably added to the redux store by mistake or for debugging. 

Could we try to set up the new state like 'state.floatingToolsVisible' or something like that, and subscribe to events from the editor to update this state? So in this case Redux will handle changes in the proper way.